app/ui.js

```diff
@@ -180,7 +180,6 @@ var UI;
             UI.initSetting('view_only', false);
             UI.initSetting('path', 'websockify');
             UI.initSetting('repeaterID', '');
-            UI.initSetting('token', '');
```

Comment:
Ah, right. I see now where it was used. However seeing that I have to object to its inclusion. Users who want a special websocket path should merely specify a different `path`. What makes this token so special?